packages/common/src/graphql/graphql.spec.ts

```diff
@@ -41,5 +40,60 @@ describe('utils that handle schema.graphql', () => {
       }
     `;
     expect(() => buildSchemaFromDocumentNode(graphqlSchema)).toThrow();
+    expect(() => buildSchemaFromDocumentNode(gql`
+      type Test @entity {
+        id: ID!
+        price: [String]
+      }
+    `)).toThrow();
+  });
+
+  it('can extract list field with entity from the schema', () => {
+    const graphqlSchema = gql`
+      type KittyBirthInfo @entity {
+        id: ID!,
+        owner: String,
+        paper: paper @derivedFrom(field: "account")
+      }
+      type paper @entity {
```

Comment:
could use a better name than paper.
and pls use capital case for type definition